octohook/events.py

```diff
@@ -552,6 +552,22 @@ def __init__(self, payload: dict):
         self.changes = _optional(payload, "changes", RawDict)
 
 
+class PullRequestReviewThreadEvent(BaseWebhookEvent):
+    """
+    https://developer.github.com/v3/activity/events/types/#pullrequestreviewthreadevent
```

Comment:
There are no docs up yet for this event, but https://github.com/google/go-github/issues/2324#issuecomment-1080659310 states that GitHub has an internal ticket to update the docs.